calisphere/templates/calisphere/facetValues.html

```diff
@@ -83,12 +83,12 @@ <h3 class="text__collections-heading">{% block objectsFound %}{{ numFound|intcom
           <div class="pag-dropdown">
 
             {% if start|subtract:rows >= 0 %}
-              <a class="js-prev btn btn-link pag-dropdown__arrow-icon" data-start="{{ start|subtract:rows }}" href="{{ form_action }}?{% if q %}q={{ q }}&{% endif %}start={{ start|subtract:rows }}"><span class="fa fa-caret-left fa-lg"></span></a>
+              <a class="js-prev btn btn-link pag-dropdown__arrow-icon" aria-label="previous set" data-start="{{ start|subtract:rows }}" href="{{ form_action }}?{% if q %}q={{ q }}&{% endif %}start={{ start|subtract:rows }}"><span class="fa fa-caret-left fa-lg"></span></a>
             {% else %}
-              <a class="js-prev btn btn-link pag-dropdown__arrow-icon" disabled><span class="fa fa-caret-left fa-lg"></span></a>
+              <button type="button" class="js-prev btn btn-link pag-dropdown__arrow-icon" aria-label="previous set" disabled><span class="fa fa-caret-left fa-lg"></span></button>
```

Comment:
Good question - yeah, this does look weird! I did this because the `disabled` HTML attribute only works with form elements, not with links. There are slightly better ways to build this pagination (so it's all links), but it looked like a significant refactor. What we have here seems to be good enough for a11y purposes.